google/cloud/storage/quickstart/README.md

```diff
@@ -160,21 +156,21 @@ Run the example, change the placeholder to appropriate values:
 
 ```bash
 bazel run :quickstart_grpc -- [BUCKET NAME]
- ```
+```
 
 ## Platform Specific Notes
 
 ### macOS
 
-gRPC [requires][grpc-roots-pem-bug] an environment variable to configure the
+gRPC \[requires\]\[grpc-roots-pem-bug\] an environment variable to configure the
```

Comment:
This changes the rendering of `[requires][grpc-roots-pem-bug]` from a link to verbatim text.

Reply:
Well, no.  The link was broken already.

Reply:
Right.  I plan to make a second pass, looking for escaped brackets as an indicator of something possibly wrong.